### ISSUE TITLE
Some WER results 

### ISSUE THREAD

**Issue by palles77** [julius-speech/julius#17]
Hello All

I spent many years working with Julius and recently performed an interesting experiment, where I manipulated various config options to get best recognition rate. I found that the best settings are available for changes related to beam "-b", insertion and deletion flags "-lmp", "-lmp2". Julius is especially sensitive to those 3 settings. You can find more details about WER and decoding times from the CSV in attachment. I hope this will help some of you.

Some of the results:
- best WER for beam = 500 is 20.50%, INS = 8, DEL = 4, Speed roughly 50 % of RT
- best WER for beam = 1000 is 13.24%, INS = 10, DEL = 0, Speed roughly 100 % of RT
- best WER for beam = 2000 is 11.91.5%, INS = 10, DEL = 0, Speed roughly 130 % of RT
- best WER for beam = 4000 is 11.22.5%, INS = 12, DEL = -4, Speed roughly  140 % of RT
- best WER for beam = 6000 is 11.06%, INS = 12, DEL = -4, Speed roughly 300 % of RT

p.s. In the attachment are detailed results plus configs for Julius.

Kind Regards
Leslaw

[TestingResults_20151229_2238.txt](https://github.com/julius-speech/julius/files/76645/TestingResults_20151229_2238.txt)
[julius_config.txt](https://github.com/julius-speech/julius/files/76648/julius_config.txt)
[wav_config.txt](https://github.com/julius-speech/julius/files/76649/wav_config.txt)

**Comment by drTonyRobinson** [comment]
Hi Leslaw,

Something swallowed all of the dots/periods in your email.  I take it 
the percentages go from 20% to 11% and the link should be 
https://github.com/julius-speech/julius/files/76645/TestingResults_20151229_2238.txt

Having said that, the same gremin will probably eat all of my 
dots/periods as well...


Tony

On 03/01/16 16:02, palles77 wrote:
>
> Hello All
>
> I spent many years working with Julius and recently performed an 
> interesting experiment, where I manipulated various config options to 
> get best recognition rate I found that the best settings are available 
> for changes related to beam "-b", insertion and deletion flags "-lmp", 
> "-lmp2" Julius is especially sensitive to those 3 settings You can 
> find more details about WER and decoding times from the CSV in 
> attachment I hope this will help some of you
>
> Some of the results:
>
>   * best WER for beam = 500 is 2050%, INS = 8, DEL = 4, Speed roughly
>     50 % of RT
>   * best WER for beam = 1000 is 1324%, INS = 10, DEL = 0, Speed
>     roughly 100 % of RT
>   * best WER for beam = 2000 is 11915%, INS = 10, DEL = 0, Speed
>     roughly 130 % of RT
>   * best WER for beam = 4000 is 11225%, INS = 12, DEL = -4, Speed
>     roughly 140 % of RT
>   * best WER for beam = 6000 is 1106%, INS = 12, DEL = -4, Speed
>     roughly 300 % of RT
>
> ps In the attachment are detailed results plus configs for Julius
>
> Kind Regards
> Leslaw
>
> TestingResults_20151229_2238txt 
> <https://githubcom/julius-speech/julius/files/76645/TestingResults_20151229_2238txt>
> julius_configtxt 
> <https://githubcom/julius-speech/julius/files/76648/julius_configtxt>
> wav_configtxt 
> <https://githubcom/julius-speech/julius/files/76649/wav_configtxt>
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/julius-speech/julius/issues/17>.
>


-- 
Speechmatics is a trading name of Cantab Research Limited
We are hiring: www.speechmatics.com/careers 
<https://www.speechmatics.com/careers>
Dr A J Robinson, Founder, Cantab Research Ltd
Phone direct: 01223 794096, office: 01223 794497
Company reg no GB 05697423, VAT reg no 925606030
51 Canterbury Street, Cambridge, CB4 3QG, UK

**Comment by palles77** [comment]
Yes - that is quite right. The best way is to look at the original post in browser and also analyse attachments. Thank you for pointing it out.
